_source/_posts/2022-08-15-spring-boot-crud-update.md

```diff
@@ -0,0 +1,1077 @@
+---
+layout: blog_post
+title: "Spring Boot CRUD app with Vue and Quasar"
+author: andrew-hughes
+by: contractor
+communities: [java,javascript]
+description: "Build a Spring Boot Todo app with a Vue client using Okta for security and the Quasar framework for client components"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+
+---
+
+You're going to use Vue and Spring Boot to build a todo list web application. The application will include CRUD abilities, meaning that you will be able to **c**reate, **r**ead, **u**pdate, and **d**elete the todo items on the Spring Boot server via the client. The Vue front-end client will use the Quasar framework for presentation. Both the Spring Boot server and the Vue client will be secured with OAuth 2.0 and OpenID Connect (OIDC) using Okta as the security provider.
+
+{% img blog/spring-boot-crud-update/spring-and-vue.png alt:"Spring Boot, Vue, and Okta logos" width:"500" %}{: .center-image }
+
+This project has two major parts:
+
+- Spring Boot server
```

Comment:
I tend to say API rather than server, but you may choose your own destiny.

Reply:
That makes sense. I'm happy either way. "server" does sound a bit like a physical server.